pages/Advanced Queries.md

```diff
@@ -408,4 +435,4 @@ description:: Advanced queries are written with [Datalog](https://en.wikipedia.o
 		- [Datomic query documentation](https://docs.datomic.com/query.html) - Thorough reference for datomic dialect of Datalog. Explains most datalog concepts well
 		- https://nextjournal.com/try/learn-xtdb-datalog-today/learn-xtdb-datalog-today - Interactive datalog tutorial for engineers
 		- [Datascript's intro docs](https://github.com/tonsky/datascript/wiki/Getting-started)
-		- [Logseq's database schema](https://github.com/logseq/logseq/blob/master/deps/db/src/logseq/db/schema.cljs)
\ No newline at end of file
+		- [Logseq's database schema](https://github.com/logseq/logseq/blob/master/deps/db/src/logseq/db/schema.cljs)
```

Comment:
And this link is broken, maybe it should point to the model file? https://github.com/logseq/logseq/blob/master/src/main/frontend/db/model.cljs

Reply:
I'm not seeing an issue with the current link